src/components/Markings/DrawingClassifier.js

```diff
@@ -141,6 +145,8 @@ class DrawingClassifier extends Component {
             return <OverlaySpinner overrideVisibility={this.props.isFetching} />
         }
 
+        // We validate that tools has at least one element earlier
+        const tool = this.props.tools[0]
```

Comment:
Is there a chance this could be null? (if used as validation, as the comment says) Also, `this.props.tools[0]` is still used in the line below, as opposed to this new variable.

Reply:
The comment is supposed to point out that tools has been validated to have `length > 0` when we validate the workflow after loading from the API